webapp/src/pages/ComponentsBook/BookSection.tsx

```diff
@@ -0,0 +1,39 @@
+import {
+  Text,
+  TextVariant,
+  TextWeight,
+  TextColor,
+} from '../../shared/components';
+
+type singleOrMultipleChildren = React.ReactElement | React.ReactElement[];
+
+type bookSectionProps = {
+  title: string;
+  children: singleOrMultipleChildren;
+};
+
+const wrapExample = (elements: singleOrMultipleChildren) =>
+  Array.isArray(elements) ? (
+    elements.map((el) => (
+      <div className="components-book__section__example-wrapper">{el}</div>
+    ))
+  ) : (
+    <div className="components-book__section__example-wrapper">{elements}</div>
+  );
+
+export const BookSection = ({ title, children }: bookSectionProps) => (
```

Comment:
This is a example section to encapsulate the title and the example content (so we just have to pass the string title and forget about styles etc)